### DISCUSSION TITLE
How to animate multiple animation in a single object

### DISCUSSION THREAD

**Discussion by aseemsalimtvm** [sceneview/sceneform-android#58] [Q&A]
val solution = "F F' U U'"
       var count = 0
      override fun onModelAdded(renderableInstance: RenderableInstance?) {
              this.renderableInstance = renderableInstance
              val move = Move.findByNotation(solution.split(" ")[0])!!
              animateMove(move,renderableInstance)
          }
      
          override fun onModelError(exception: Throwable?) {
      
          }

    private fun animateMove(move : Move, renderableInstance: RenderableInstance?){
        if(renderableInstance == null) return
        val animation = when(move){
            Move.F-> renderableInstance.animate(
                "F_00",
                "F_01",
                "F_10",
                "F_11"
            )
            Move.F_PRIME-> renderableInstance.animate(
                "F'_00",
                "F'_01",
                "F'_10",
                "F'_11"
            )
            Move.U-> renderableInstance.animate(
                "U_00",
                "U_01",
                "U_10",
                "U_11"
            )
            Move.U_PRIME-> renderableInstance.animate(
                "U'_00",
                "U'_01",
                "U'_10",
                "U'_11"
            )
            else -> null
        }
        animation?.apply {
            duration = 2000
            repeatCount = 0
            addListener(this@SceneFormArActivity)
            start()
        }
    }

    override fun onAnimationStart(animation: Animator?) {

    }

    override fun onAnimationEnd(animation: Animator?){
        count++
        if(count < 4) {
            val move = Move.findByNotation(solution.split(" ")[count])!!
            Log.d(TAG, "onAnimationEnd: Move = ${move.notation}")
            animateMove(
                move,
                renderableInstance
            )
        }
    }

i want to perform series of animations but some of the animation not animating , here   
"F_00",
                "F_01",
                "F_10",
                "F_11" 
 "F'_00",
                "F'_01",
                "F'_10",
                "F'_11"
are the animation of 4 objects in 2 different moves in first Move Move.F first four animations get played after that i want play the reverse animation using the Move.F_PRIME but nothing playing

**Comment by ThomasGorisse** [comment]
If you want to play multiple animations on the same object at same time, use the `setAutoCancel(false)`
